client/src/pages/WeeklyTable.js

```diff
@@ -1,19 +1,19 @@
 import React from "react";
 import { useState } from "react";
-
+import moment from "moment";
 import { formatBookingDate } from "../util";
 import ModalBookingScreen from "./ModalBookingScreen";
 
 import "./WeeklyTable.css";
 // const DEFAULT_MONDAY = new Date(2022, 0, 17);
 
-const week = [
-	"2022-01-17T00:00:00.000Z",
-	"2022-01-18T00:00:00.000Z",
-	"2022-01-19T00:00:00.000Z",
-	"2022-01-20T00:00:00.000Z",
-	"2022-01-21T00:00:00.000Z",
-];
+// const week = [
+// 	"2022-01-17T00:00:00.000Z",
+// 	"2022-01-18T00:00:00.000Z",
+// 	"2022-01-19T00:00:00.000Z",
+// 	"2022-01-20T00:00:00.000Z",
+// 	"2022-01-21T00:00:00.000Z",
+// ];
```

Comment:
Why not deleting the comments to keep the code-base cleaner?

Reply:
done, many thanks for reviewing